docs/source/hooks.rst

```diff
@@ -40,15 +40,13 @@ Training loop
 ^^^^^^^^^^^^^
 
 - :meth:`~pytorch_lightning.core.hooks.ModelHooks.on_epoch_start`
-- :meth:`~pytorch_lightning.core.hooks.ModelHooks.on_batch_start`
```

Comment:
shall we add `on_train_batch_start`?

Reply:
Good point, mind make it as suggestion here?

Reply:
Github shows `Applying suggestions on deleted line is not supported.`
```
- :meth:`~pytorch_lightning.core.hooks.ModelHooks.on_train_batch_start`
```